### PR TITLE
Wire up "Request Individual Mapper Status" button on group detail page

### PR DESCRIPTION
**Pull request by kdeloach**
How to test:
1. Select a group that allows individual mappers
2. Login as a non-group admin that is not a trusted mapper (for your test group)
3. Go to the group detail page
4. You should see a "Request Individual Mapper" button
5. Clicking this button should create a new mapper request in the Group Admin page
6. When a group admin approves this request, the button should disappear from the group detail page

### CODE REVIEW

File: src/nyc_trees/js/src/group_detail.js [hunk 1]
```diff
@@ -16,3 +24,20 @@ var territoryMap = mapModule.create({
 });
 
 mapModule.addTileLayer(territoryMap);
+
+$(dom.requestAccessButton).click(function() {
+    var groupSlug = $(dom.requestAccessButton).data('group-slug');
+    $(dom.modals).modal('hide');
+    $.ajax({
+            // Keep this URL in sync with "request_mapper_status"
+            // in src/nyc_trees/apps/users/urls/group.py
```
Comment (maurizi) [inline src/nyc_trees/js/src/group_detail.js]: Consider adding to the existing comment in the referenced file

**Comment by maurizi** [comment]
Other than my 1 comment, this looks good.  Testing it out now.

File: src/nyc_trees/js/src/group_detail.js (hunk 1 above)
Comment (kdeloach) [inline src/nyc_trees/js/src/group_detail.js]: Done.

**Comment by maurizi** [comment]
+1, works

**Comment by RickMohr** [comment]
+1 after fixing lint

**Pull request merged**